src/exchange.ts

```diff
@@ -32,6 +30,7 @@ export const devtoolsExchange: Exchange = ({ client, forward }) => {
 
   // Listen for messages from content script
   window.addEventListener(DevtoolsExchangeIncomingEventType, event => {
+    console.log(event);
```

Comment:
Redundant log

Reply:
Yeah hold of on this for a short mo - needs some tweaks